dali/python/nvidia/dali/pipeline.py

```diff
@@ -722,7 +722,9 @@ def __exit__(self, type, value, traceback):
     def _require_unique_names(self):
         ops_by_name = {}
         for op in self._ops:
-            ops = ops_by_name.get(op.name, [])
+            ops = ops_by_name.get(op.name, None)
+            if ops is None:
+                ops = ops_by_name[op.name] = []
```

Comment:
This made the test always pass (the dictionary was empty). There was another check in C++, so it's not critical.